### PR TITLE
Update label on modeling.updateModdleProperties

### PR DESCRIPTION
**Pull request by barmac**
Closes https://github.com/bpmn-io/bpmn-js/issues/1872

I extracted one of the bugs into a separate issue: https://github.com/bpmn-io/bpmn-js/issues/1893

### CODE REVIEW

File: lib/features/label-editing/cmd/UpdateLabelHandler.js [hunk 1]
```diff
@@ -77,7 +53,7 @@ export default function UpdateLabelHandler(modeling, textRenderer, bpmnFactory)
 
     setLabel(label, text, labelTarget !== label);
 
-    ensureInternalLabelDi(element, text);
+    // ensureInternalLabelDi(element, text);
```
Comment (nikku) [inline lib/features/label-editing/cmd/UpdateLabelHandler.js:56]: We'd want to get rid of this line, right?

File: lib/features/modeling/BpmnUpdater.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ import {
 
 import { isAny } from './util/ModelingUtil';
 
-import { isLabel } from '../../util/LabelUtil';
+import {
+  isLabel,
+  isLabelExternal
+} from '../../util/LabelUtil';
```
Comment (nikku) [inline lib/features/modeling/BpmnUpdater.js:24]: Combine both imports.

File: lib/features/modeling/behavior/LabelBehavior.js [hunk 3]
```diff
@@ -86,7 +79,16 @@ export default function LabelBehavior(
   CommandInterceptor.call(this, eventBus);
 
   // update label if name property was updated
-  this.postExecute('element.updateProperties', function(e) {
+  this.postExecute('element.updateProperties', onPropertyUpdate);
```
Comment (nikku) [inline lib/features/modeling/behavior/LabelBehavior.js:82]: Do we also cover text annotation and groups (category) here? Is that needed?

**Review by nikku — CHANGES_REQUESTED** [review]
Nice one! Please checkout my remarks.

File: lib/features/label-editing/cmd/UpdateLabelHandler.js (hunk 1 above)
Comment (barmac) [reply lib/features/label-editing/cmd/UpdateLabelHandler.js:56]: Yeah, my bad

File: lib/features/modeling/BpmnUpdater.js (hunk 2 above)
Comment (barmac) [reply lib/features/modeling/BpmnUpdater.js:25]: I moved LabelUtil up while maintaining backwards compatibility: https://github.com/bpmn-io/bpmn-js/pull/1894/commits/b05649220b2206956dd8acd113f01d7f3de472f0

File: lib/features/modeling/behavior/LabelBehavior.js (hunk 3 above)
Comment (barmac) [reply lib/features/modeling/behavior/LabelBehavior.js:79]: As you can see below, TextAnnotation is still covered. Not sure about the groups. However, I don't want to creep the scope of this PR.

File: lib/features/modeling/behavior/LabelBehavior.js [hunk 4]
```diff
@@ -133,20 +131,7 @@ export default function LabelBehavior(
       return;
     }
 
-    var labelCenter = getExternalLabelMid(element);
-
-    // we don't care about x and y
-    var labelDimensions = textRenderer.getExternalLabelBounds(
-      DEFAULT_LABEL_DIMENSIONS,
```
Comment (nikku) [inline lib/features/modeling/behavior/LabelBehavior.js:140]: Where did the `DEFAULT_LABEL_DIMENSIONS` go? We don't need it anymore?

**Review by nikku — APPROVED** [review]
Good improvement and refactoring :tada:

**Pull request merged**